README_INTRO.md

```diff
@@ -0,0 +1,16 @@
+# StarPlayer
+# A player to learn from!  #
+
+**This is a open source music player applciation for learning many of libraries such as coroutines, RxJava, Dagger and ...**
+
+We will be working on it with to learn new things as has been introduced as a good things in android worlds.
```

Comment:
.... on it to learn ....

.... as good things ....

.... in android world ....